### DISCUSSION TITLE
"unicorns have taken over" Error - 503 -

### DISCUSSION THREAD

**Discussion by varanasisaigithub** [community/community#23451] [Other Feature Feedback, Questions, & Ideas]
<p>after a recent change  on site - <a href="https://transrev.github.io/" rel="noopener nofollow ugc">https://transrev.github.io/</a>  reports 503 error.</p>
<p>I have checked <code>actions</code> and page build, deploy was successful.</p>

**Comment by tcbyrd** [comment]
<p>Hey there <a class="mention" href="/varanasisaigithub">@varanasisaigithub</a>, and welcome to the community forum! Sorry about this. We had a bug in the deployment process that only effects sites that haven’t built in a while. While the deploy did succeed, we were still attempting to route to the site as if it was built using an older method. We’ve fixed the bug and triggered a rebuild in your site to get it working. Any new builds shouldn’t have this problem.</p>

**Comment by steezeburger** [comment]
<p><a class="mention" href="/tcbyrd">@tcbyrd</a> i seem to be having the same issue with <a href="http://steezeburger.github.io/conway" rel="noopener nofollow ugc">http://steezeburger.github.io/conway</a></p>

**Comment by queenofcorgis** [comment]
<p><a class="mention" href="/steezeburger">@steezeburger</a> have you tried triggering a rebuild?</p>

**Comment by steezeburger** [comment]
<p><a class="mention" href="/liligalante">@liligalante</a> i did! it was successful as well, but I still see the unicorn error page when i visit the site!</p>

**Comment by queenofcorgis** [comment]
<p>Hmmm :thinking: is this site a new build? <a class="mention" href="/tcbyrd">@tcbyrd</a> any ideas as well?</p>

**Comment by steezeburger** [comment]
<p><a class="mention" href="/liligalante">@liligalante</a> it’s a pretty old repo. I hadn’t touched it in several years before today!</p>

**Comment by ryanspradlin** [comment]
<p>Hi <a class="mention" href="/liligalante">@liligalante</a> and <a class="mention" href="/tcbyrd">@tcbyrd</a>, I seem to have encountered the same issue with <a href="https://ryanspradlin.github.io" rel="noopener nofollow ugc">https://ryanspradlin.github.io</a>. Old repo, last change was in 2016, just pushed a change today, and now get HTTP 503 responses. I tried a rebuild, no luck. Help would be much appreciated. Thanks!</p>

**Comment by tcbyrd** [comment]
<p><a class="mention" href="/ryanspradlin">@ryanspradlin</a> <a class="mention" href="/steezeburger">@steezeburger</a> Your pages have been fixed. In case anyone else comes here and sees this, a workaround for now is to disable Pages on your repository and re-enable it. We’re working on fixing the bug that causes this.</p>

**Comment by ryanspradlin** [comment]
<p><a class="mention" href="/tcbyrd">@tcbyrd</a> Thank you for the quick fix!</p>

**Comment by steezeburger** [comment]
<p><a class="mention" href="/tcbyrd">@tcbyrd</a> thanks so much! You rock!</p>

**Comment by tcbyrd** [comment, marked as answer]
<p>We’ve shipped this fix that caused this and it shouldn’t require a rebuild or disabling/enabling Pages to fix it from here on out. If you were previously seeing a 503 and didn’t get a chance to rebuild, the page should resolve properly now. If anyone is still seeing issues, feel free to let us know though so we can dig in further.</p>

**Comment by kamiyaa** [comment]
This is still happening for github page links such as https://clkliondance.com/_app/immutable/chunks/SidePadding-11688f4b.js

**Comment by legen07** [comment]
> <p>We’ve shipped this fix that caused this and it shouldn’t require a rebuild or disabling/enabling Pages to fix it from here on out. If you were previously seeing a 503 and didn’t get a chance to rebuild, the page should resolve properly now. If anyone is still seeing issues, feel free to let us know though so we can dig in further.</p>

<a href="hugeone.store"> I am also g etting same issue hugeone.store </a>

**Comment by conveypride** [comment]
I'm getting tthe same error on my site;
[Conveypride ](https://conveypride.github.io/) , and also profile images not showing

**Comment by Anamaruchi** [comment]
getting some error to my pages
https://anamaruchi.github.io/ already unpublish and republish but still same

**Reply by idindrakusuma** [reply]
I got same issue too. I created the discussion here https://github.com/orgs/community/discussions/180097

**Reply by mzy0624** [reply]
Same [https://mzy0624.github.io/scripts/popup.js](https://mzy0624.github.io/scripts/popup.js)

**Reply by AngelliaX** [reply]
kinda weird that it only affect 1 specific route of the entire github page
